extensions/spotify-player/src/api/startRadio.ts

```diff
@@ -21,11 +21,7 @@ export async function startRadio({ trackIds = [], artistIds = [] }: StartRadioPr
       } catch (err) {
         const error = getErrorMessage(err);
         if (error?.toLocaleLowerCase().includes("no active device")) {
-          const script = buildScriptEnsuringSpotifyIsRunning(`tell application "Spotify"
-          launch
-          delay 3
-          play track "${tracks[0].uri}"
-  end tell`);
```

Comment:
The `buildScriptEnsuringSpotifyIsRunning` function already handles all of the launching Spotify logic.